routes/api/gameApi.js

```diff
@@ -3,10 +3,8 @@ const router = require("express").Router();
 const UserGame = require("../../models/UserGame");
 const Games = require("../../models/Games");
 const Users = require("../../models/Users");
-const GameCollection = require("../../models/GameCollection");
```

Comment:
Didn't we remove these in the other PR?

Reply:
Thought so, but when i pulled main, it was there and and i just followed the errors till it run again
![Screen Shot 2021-06-08 at 11 04 10 am](https://user-images.githubusercontent.com/78417151/121108030-60adba80-c84c-11eb-8ee7-c947a950ca55.png)